data/data_group_2.json

```diff
@@ -11,7 +11,7 @@
     },
     {
       "group_name": "Frame 2",
-      "cycle_time_ms": 20,
+      "cycle_time_microseconds": 20,
```

Comment:
Just want to make sure that it is intentional to change semantic meaning of the example, i.e. that we in some cases (like here) keep the numeric value, in other multiply be 100

Reply:
It was actually non intentional, just left some value, provided more realistic values